bool detectLeft
float64 angleLeft
float64 distanceLeft
bool detectRight
float64 angleRight
float64 distanceRight
bool detectTop
float64 angleTop
float64 angleBot
